### PR TITLE
Document permissions that do not require allowlisting

### LINKED ISSUE #1309: Snaps: Update Get Allowlisted to mention Snaps that do not require allowlisting and list permissions

### PR DESCRIPTION
**Pull request by Montoya**
# Description

Updated Snaps Get Allowlisted page

## Issue(s) fixed

Fixes #1309

## Checklist

Complete this checklist before merging your PR:

- [x] If this PR contains a major change to the documentation content, I have added an entry to the top of the ["What's new?"](https://github.com/MetaMask/metamask-docs/blob/main/docs/whats-new.md) page.
- [ ] The proposed changes have been reviewed and approved by a member of the documentation team.

### CODE REVIEW

**Comment by Montoya** [comment]
This is a WIP. I am asking the content design team to help me come up with the right messaging for this.

**Comment by Montoya** [comment]
OK content design has suggested _protected_ vs _open_ permissions

**Review by eriknson — APPROVED** [review]

File: docs/whats-new.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ of the [MetaMask developer page](https://metamask.io/developer/).
 
 ## May 2024
 
+- Updated [Snaps Allowlisting Guide](/snaps/how-to/get-allowlisted.md#open-permissions) with "Open Permissions."
```
Comment (alexandratran) [inline docs/whats-new.md:14]:
```suggestion
- Updated [Snaps allowlisting guide](/snaps/how-to/get-allowlisted.md#open-permissions) with open permissions.
```

**Review by hmalik88 — APPROVED** [review]

File: snaps/how-to/get-allowlisted.md [hunk 2]
```diff
@@ -5,11 +5,13 @@ sidebar_position: 8
...
-MetaMask users.
+Once you have built your Snap, tested it, and published it to npm, you can make it available to MetaMask users.
 
-As part of the MetaMask Snaps Open Beta launch, individual Snaps must be put on an allowlist before users can install them.
-This means that at this time, only selected, reviewed Snaps can be installed. 
+If your Snap only uses [open permissions](#open-permissions), it can be installed by anyone on the MetaMask Extension. 
+However, individual Snaps that use 
+[protected permissions](#open-permissions) 
+must be put on an allowlist before users can install them.
+This means that at this time, only selected, reviewed Snaps that use protected permissions can be installed.
```
Comment (alexandratran) [inline snaps/how-to/get-allowlisted.md:14]:
```suggestion
If your Snap only uses [open permissions](#open-permissions), anyone can install it on the MetaMask extension. 
However, Snaps that use 
[protected permissions](#open-permissions) 
must be put on an allowlist before users can install them.
This means that at this time, for Snaps that use protected permissions, only those that are selected and reviewed can be installed. 
```

File: snaps/how-to/get-allowlisted.md [hunk 3]
```diff
@@ -145,3 +147,23 @@ You can also update any information about your Snap using the form.
...
+- `endowment:cronjob`
+- `endowment:ethereum-provider`
+- `endowment:lifecycle-hooks`
+- `endowment:page-home`
+- `endowment:signature-insight`
+- `endowment:transaction-insight`
+- `snap_dialog`
+- `snap_getLocale`
+- `snap_manageState`
+- `snap_notify`
```
Comment (alexandratran) [inline snaps/how-to/get-allowlisted.md:164]:
```suggestion
## Open permissions

The following is a list of permissions that do not require allowlisting: 

- [`endowment:cronjob`](../reference/permissions.md#endowmentcronjob)
- [`endowment:ethereum-provider`](../reference/permissions.md#endowmentethereum-provider)
- [`endowment:lifecycle-hooks`](../reference/permissions.md#endowmentlifecycle-hooks)
- [`endowment:page-home`](../reference/permissions.md#endowmentpage-home)
- [`endowment:signature-insight`](../reference/permissions.md#endowmentsignature-insight)
- [`endowment:transaction-insight`](../reference/permissions.md#endowmenttransaction-insight)
- [`snap_dialog`](../reference/snaps-api.md#snap_dialog)
- [`snap_getLocale`](../reference/snaps-api.md#snap_getlocale)
- [`snap_manageState`](../reference/snaps-api.md#snap_managestate)
- [`snap_notify`](../reference/snaps-api.md#snap_notify)
```

File: snaps/how-to/get-allowlisted.md [hunk 4]
```diff
@@ -145,3 +147,23 @@ You can also update any information about your Snap using the form.
...
+
+If your Snap only uses permissions from this list, 
+it can be installed in the MetaMask extension without inclusion on the allowlist. 
+
+Any permissions not on this list are _protected_ permissions and will require allowlisting.
```
Comment (alexandratran) [inline snaps/how-to/get-allowlisted.md:169]:
```suggestion
Any permissions not on this list are _protected permissions_ and require allowlisting. 
```

File: docs/whats-new.md [hunk 5]
```diff
@@ -11,6 +11,7 @@ of the [MetaMask developer page](https://metamask.io/developer/).
 
 ## May 2024
 
+- Updated [Snaps allowlisting guide](/snaps/how-to/get-allowlisted.md#open-permissions) with open permissions.
```
Comment (alexandratran) [inline docs/whats-new.md:14]:
```suggestion
- Updated [Snaps allowlisting guide](/snaps/how-to/get-allowlisted.md#open-permissions) with open permissions.
  ([#1310](https://github.com/MetaMask/metamask-docs/pull/1310/))
```

**Review by alexandratran — APPROVED** [review]

**Pull request merged**